app/controller/controller_users.py

```diff
@@ -4,7 +4,6 @@
 from ..models import User, user_schema
 from werkzeug.security import generate_password_hash, check_password_hash
 from ..ext.api_avatar import create_avatar
-from flask import jsonify, make_response
 from config import Config
```

Comment:
Please remove unsused imports. Use IDE to help you in that. they would have a greyed color